mitodl/mitxonline
# Add program_type field to database

# What are the relevant tickets?

Closes mitodl/hq#1982

# Description (What does it do?)

Adds a `program_type` field to the Program model and defaults it to `Series`. This is part of the new lite redesign - this field is needed elsewhere and having it in the current design won't hurt anything, so going forward with adding it now.

This also changes a couple migrations to use the versioned app models for `Program`. The affected migrations were directly importing the model, which ends up having a field that hasn't been applied yet and thus causes issues. This is really only an issue if you're spinning up a new instance or for CI; existing databases should have the migrations applied already and there's no functional change to them. 

# How can this be tested?

Run the migration. Afterwards, you should see a "Program type" field in the Django admin.
